stubs/pyflakes/METADATA.toml

```
@@ -1,2 +1,2 @@
-version = "~=3.3.2"
+version = "~=3.4.0"
```

Could simplify now that this doesn't need to exclude any patch releases:
```suggestion
version = "3.4.*"
```